### PR TITLE
Have done task for lesson-11

### PR DESCRIPTION
**Pull request by targonskii**
Added filter for adults, children and rooms.

### CODE REVIEW

File: lesson-11/main.js [hunk 1]
```diff
@@ -1,43 +1,96 @@
...
-const focusPeople = (e) => {
-    peopleFilter.style.display = 'block';
-    peopleInput.setAttribute('placeholder', '0 Adults — 0 Children — 0 Room')
-}
-peopleInput.addEventListener('focus', focusPeople);
+const focusPeople = () => {
+  peopleFilter.style.display = 'block';
+  adultsInput.setAttribute('placeholder', '0 Adults     — ');
+  childrenInput.setAttribute('placeholder', '0 Children   — ');
+  roomsInput.setAttribute('placeholder', '0 Room');
```
Comment (nikitik-nikita) [inline lesson-11/main.js]: This code is unnecessary. We should change initial input value

File: lesson-11/main.js [hunk 2]
```diff
@@ -1,43 +1,96 @@
...
+};
```
Comment (nikitik-nikita) [inline lesson-11/main.js]: You can also add condition like if 
```
(event.target === adultsInput) { 
  peopleFilter.style.display = 'block'; 
} else { 
  peopleFilter.style.display = 'none'; 
}
```

**Review by nikitik-nikita — COMMENTED** [review]
1. You have `label for="Children"` and you also need to add the same id for input in your html file (fix this for all labels)
2. Try to add disabled styles to buttons when the value from the input reaches a certain condition

File: lesson-11/main.js (hunk 1 above)
Comment (targonskii) [reply lesson-11/main.js]: Deleted this code.

File: lesson-11/main.js (hunk 2 above)
Comment (targonskii) [reply lesson-11/main.js]: Added conditions.

File: lesson-11/main.js [hunk 3]
```diff
@@ -1,43 +1,127 @@
...
+  if (e.target === adultsInput || childrenInput || roomsInput) {
     peopleFilter.style.display = 'block';
-    peopleInput.setAttribute('placeholder', '0 Adults — 0 Children — 0 Room')
-}
-peopleInput.addEventListener('focus', focusPeople);
+  } else {
+    peopleFilter.style.display = 'none';
+  }
+};
+peopleInput.addEventListener('click', focusPeople);
```
Comment (nikitik-nikita) [inline lesson-11/main.js:10]: lets change `peopleInput` to `window`

File: lesson-11/main.js [hunk 4]
```diff
@@ -1,43 +1,127 @@
-const peopleInput = document.querySelector('.header__people input');
+const peopleInput = document.querySelector('.header__people');
+const adultsInput = document.querySelector('.header__input_adults');
+const childrenInput = document.querySelector('.header__input_children');
+const roomsInput = document.querySelector('.header__input_rooms');
 const peopleFilter = document.querySelector('.header__people_filter');
 
 const focusPeople = (e) => {
+  if (e.target === adultsInput || childrenInput || roomsInput) {
```
Comment (nikitik-nikita) [inline lesson-11/main.js]: `(e.target === adultsInput || e.target === childrenInput || e.target === roomsInput)`

**Review by nikitik-nikita — COMMENTED** [review]
If you click on the button to add children more than 10 times, the button disables, but the rendering of filters with age continues

**Comment by targonskii** [comment]
> If you click on the button to add children more than 10 times, the button disables, but the rendering of filters with age continues

Fixed this trouble.
But still have problem with window.

**Comment by targonskii** [comment]
Added method toggle for filter.

**Review by nikitik-nikita — COMMENTED** [review]
nit: Let's shorten your code a bit. we can create variables for repeated code (e.g. e.target.parentElement.querySelector('.header__people_adults input'), minus.style.color, plus.style.color, etc.).
The code will be shorter and prettier

**Comment by targonskii** [comment]
> nit: Let's change the name of the class to something more appropriate (e.g. visible)

Fixed it

**Comment by targonskii** [comment]
> nit: Let's shorten your code a bit. we can create variables for repeated code (e.g. e.target.parentElement.querySelector('.header__people_adults input'), minus.style.color, plus.style.color, etc.).
> The code will be shorter and prettier

Done, pls check.

**Pull request merged**